src/react-components/src/components/01-atoms/Images/Icons/Icon.tsx

```diff
@@ -1,46 +1,57 @@
-
 import * as React from "react";
-import Icons from "@nypl/design-system-icons/dist/sprite/sprite.svg";
 import bem from "../../../../utils/bem";
 
-  // Wrapper Class for Icon
-  export interface IconProps {
-    name: string;
-    blockName?: string;
-    modifiers?: string[];
-    decorative: boolean;
-    role?: string;
-    title?: boolean;
-    desc?: boolean;
-  }
+import arrow_xsmall from "../../../../../icons/svg/arrow-xsmall.svg";
+import arrow from "../../../../../icons/svg/arrow.svg";
+import download from "../../../../../icons/svg/download.svg";
+import minus from "../../../../../icons/svg/minus.svg";
+import plus from "../../../../../icons/svg/plus.svg";
+import search_small from "../../../../../icons/svg/search-small.svg";
+import search from "../../../../../icons/svg/search.svg";
 
-  export default class Icon extends React.Component<IconProps, {}> {
-    constructor(props: IconProps) {
-      super(props);
-    }
+const allSvgs = {
+  arrow_xsmall: arrow_xsmall,
+  arrow: arrow,
+  download: download,
+  minus: minus,
+  plus: plus,
+  search_small:
+  search_small,
+  search: search
+};
 
-    render(): JSX.Element {
-      let { modifiers, blockName, name, decorative, role, title, desc } = this.props;
-      let icon_base_class = "icon";
+// Wrapper Class for Icon
+export interface IconProps {
+  name: string;
+  blockName?: string;
+  modifiers?: string[];
+  decorative: boolean;
+  role?: string;
+  title?: boolean;
+  desc?: boolean;
+}
+
+export default class Icon extends React.Component<IconProps, {}> {
+  constructor(props: IconProps) {
+    super(props);
+  }
 
-      let iconProps = {
-        "className": bem(icon_base_class, modifiers, blockName),
-        "role": decorative ? "img" : role,
-        "aria-hidden": decorative,
-        "aria-labelledby": title ? "title-" + name : undefined,
-        "aria-describedby": desc ? "desc-" + name : undefined
-      };
+  render(): JSX.Element {
+    let { modifiers, blockName, name, decorative, role, title, desc } = this.props;
+    let icon_base_class = "icon";
 
-      return (
-        <svg {...iconProps} >
-            {title &&
-              <title id={`title-${name}`}>{ title }</title>
-            }
-            {desc &&
-              <desc id={`title-${name}`}>{ desc }</desc>
-            }
-            <use href={`${Icons}#${name}`} />
-        </svg>
-      );
-    }
+    let iconProps = {
+      "className": bem(icon_base_class, modifiers, blockName),
+      "role": decorative ? "img" : role,
+      "aria-hidden": decorative,
+      "aria-labelledby": title ? "title-" + name : undefined,
+      "aria-describedby": desc ? "desc-" + name : undefined,
+      "title": title ? `title-${name}` : undefined,
+      "description": desc ? `title-${name}` : undefined
+    };
+    let svg = allSvgs[name];
+    return (
+      <div {...iconProps} dangerouslySetInnerHTML={{ __html: svg }} />
```

Comment:
@scooooooooby is this going to cause problems?  The svg is passed as a string and not natively parseable, so it's a bit more complicated to get `<svg {...iconProps}></svg>` but I can look into it more if this is a thing we definitely need.

Reply:
Perhaps in the future (say, the UX team wants to begin animating icons, or dynamically changing their fills based on component state) but we haven't received any designs with that sort of thing yet. For now I don't think we need to overengineer this and this is great!

Reply:
Sounds good.  It's also not a problem that we're no longer using the spritesheet, right?  I will remove the unused scripts if that's ok

Reply:
Yeah, I think now that you can call `Icon` and send it a name, that seems good to me